### PR TITLE
Feature/submit new show

### PR DESCRIPTION
**Pull request by davidleach724**
# Description
This feature saves the new show/ campsite information to state so that it will be displayed with the other shows

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactor Code

## How Has This Been Tested?
- [x] npm start
- [x] console.log()
- [x] dev tools
- [ ] cypress testing

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

## Additional Comments for Team Members if Applicable 
Please include a summary if applicable

### CODE REVIEW

**Comment by davidleach724** [comment]
You can tag people like this?? @Meggs625

File: src/components/AddDate/AddDate.js [hunk 1]
```diff
@@ -29,29 +36,48 @@ const AddDate = () => {
         <div className="left-form">
           <input
             type='date'
+            onChange={e => setNewShow({...newShow, date: e.target.value})}
             />
           <input
             type='text'
             placeholder='Venue'
             name='venue'
+            onChange={e => setNewShow({...newShow, venue: e.target.value})}
```
Comment (Meggs625) [inline src/components/AddDate/AddDate.js:45]: Ooo, that's a really cool onChange

File: src/components/App/App.js [hunk 2]
```diff
@@ -1,20 +1,32 @@
...
+import React, { useEffect, useState } from 'react';
 import { Route, Switch } from 'react-router';
 import CurrentTour from '../CurrentTour/CurrentTour';
 import NavBar from '../NavBar/NavBar';
 import AddDate from '../AddDate/AddDate';
+import { userMockData } from '../../dataFiles/userData'
 import './App.css';
 
 const App = () => {
+  const [bookedShows, setBookedShows] = useState([])
```
Comment (Meggs625) [inline src/components/App/App.js:10]: Sweet hooks

File: src/components/App/App.js [hunk 3]
```diff
@@ -1,20 +1,32 @@
...
+
   return (
     <main className="App">
       <NavBar />
       <Switch>
-        <Route exact path='/currentTour/' component={ CurrentTour } />
-        <Route exact path='/addDate/' component={ AddDate } />
+        {bookedShows &&
+        <Route exact path='/currentTour/' render={() => <CurrentTour userMockData={bookedShows}/>} />}
+        <Route exact path='/addDate/' render={() => <AddDate updateShows={updateShows}/>} />
```
Comment (Meggs625) [inline src/components/App/App.js:26]: Clean Switch - Going to add an catch all as default?

**Review by Meggs625 — APPROVED** [review]
Wow, this is looking great! You've put a lot of work into this - well done!

File: src/components/App/App.js (hunk 3 above)
Comment (davidleach724) [reply src/components/App/App.js:26]: Absolutely! Thanks for the reminder

**Pull request merged**